extensions/amp-fit-text/amp-fit-text.md

```diff
@@ -2,15 +2,14 @@
 $category@: presentation
 formats:
   - websites
-  - stories
-  - ads
-  - email
 teaser:
   text: Expands or shrinks font size to fit the content within the space given.
+experimental: true
```

Comment:
Hah! Forgot about this flag. This solves the problem we discussed in our meeting that the component will be shown as default.